node/network/approval-distribution/src/lib.rs

```diff
@@ -1758,3 +1733,59 @@ impl<Context> ApprovalDistribution {
 		SpawnedSubsystem { name: "approval-distribution-subsystem", future }
 	}
 }
+
+/// The maximum amount of assignments per batch is 33% of maximum allowed by protocol.
+/// This is an arbitrary value. Bumping this up increases the maximum amount of approvals or assignments
+/// we send in a single message to peers. Exceeding `MAX_NOTIFICATION_SIZE` will violate the protocol
+/// configuration.
+pub const MAX_BATCH_SIZE: usize = MAX_NOTIFICATION_SIZE as usize /
+	std::mem::size_of::<(IndirectAssignmentCert, CandidateIndex)>() /
+	3;
```

Comment:
Would be safer if we guaranteed via `min` that it is at least as large as a single message.

Reply:
Why is the size the same for approvals and assignments. I would assume that assignments are larger, so you just picked the larger one. ... Still just having two constants here would be hardly more work, but more correct and a bit more robust against future changes.

Reply:
That's fair. Initially I had 2 constants, but Idecided that it would be simpler with one, without compromising on anything else. I'll change it back to 2 constants and also provide the said guarantee.

Reply:
done